mkdocs.yml

```diff
@@ -136,16 +146,10 @@ nav:
     - examples/piecewise_constraints.py
     - examples/calliope_model_object.py
     - examples/calliope_logging.py
-  - Advanced features:
-    - advanced/constraints.md
-    - advanced/index.md
-    - advanced/time.md
-    - advanced/scripts.md
-    - advanced/solver.md
-    - advanced/backend_choice.md
-    - advanced/backend_interface.md
-    - advanced/shadow_prices.md
+    - Example additional math gallery:
```

Comment:
when clicking through, this is a bit hidden in the list. Would it be info overload to have these at the same level as the rest of the examples? We could improve the titles to being like "running with XX" or "adding XX to your model"?